ultipro/info.json

```
@@ -0,0 +1,385 @@
+            "output_schema": {},
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "Employee ID",
+                    "type": "text",
+                    "name": "employeeId",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": ""
+                }
+            ],
+            "open": false
+        },
+        {
+            "operation": "findEmployeePhone",
+            "annotation": "findEmployeePhone",
+            "title": "Find Employee Phones",
+            "description": "This method will return all employee phones with the Company Code specified in the connector configuration",
+            "output_schema": {},
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "PageNumber",
+                    "type": "text",
+                    "name": "pageNumber",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1"
+                },
+                {
+                    "title": "Page Size",
+                    "type": "text",
+                    "name": "pageSize",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1000"
+                }
+            ],
+            "open": false
+        },
+        {
+            "operation": "getAllCompanyEmployees",
+            "annotation": "getAllCompanyEmployees",
+            "title": "Get All Employees",
+            "description": "This method will return all employees with the Company Code specified in the connector configuration",
+            "output_schema": {},
+            "enabled": true,
+            "parameters": [
+                {
+                    "title": "PageNumber",
+                    "type": "text",
+                    "name": "pageNumber",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1"
+                },
+                {
+                    "title": "Page Size",
+                    "type": "text",
+                    "name": "pageSize",
+                    "required": true,
+                    "visible": true,
+                    "editable": true,
+                    "value": "1000"
+                }
+            ],
+            "open": false
+        },
+        {
+            "operation": "getPersonDetails",
+            "annotation": "getPersonDetails",
+            "title": "Get Person Details",
+            "description": "This method allows you to retrieve an individual persons details for a single company. ",
+            "output_schema": {
+                "response": [
+                    {
+                        "additionalName1": "string",
```

```suggestion
                        "additionalName1": "",
```
For output schema please keep value as empty string as shown above.